lib/chef/knife/ec_backup.rb

```diff
@@ -91,11 +98,13 @@ def for_each_user
 
       def for_each_organization
         rest.get('/organizations').each_pair do |name, url|
+          $organizations_hash["total"]=$organizations_hash["total"]+1
```

Comment:
```suggestion
          $organizations_hash["total"]+=1
```
Shorter and more idiomatic.